src/main/java/com/tlswe/awsmock/ec2/control/MockEC2QueryHandler.java

```diff
@@ -2048,4 +2054,19 @@ private String getBlankResponseXml() {
         }
         return ret;
     }
+
+    /**
+     * Gets the VPC id for a given subnetId.
+     *
+     * @param subnetId The subnet id.
+     * @return The VPC id. Returns null, if no matching subnet is found.
+     */
+    private String getVpcForSubnetId(final String subnetId) {
```

Comment:
It would be more meaningful if we change the name to getVpcIdForSubnet

Reply:
Agreed